format-all.el

```diff
@@ -623,6 +624,12 @@ Consult the existing formatters for examples of BODY."
   (:languages "PureScript")
   (:format (format-all--buffer-easy executable "-")))
 
+(define-format-all-formatter refmt
+  (:executable "bsrefmt")
+  (:install "npm install --global bs-platform")
+  (:languages "ReasonML")
```

Comment:
Please change this from `"ReasonML"` to `"Reason"`. `"Reason"` is recognized by the GitHub Linguist project, so I added it to the Emacs [language-id](https://github.com/lassik/emacs-language-id) package which format-all uses.

Reply:
That's good to know :-)